client/sites/components/sites-dashboard.tsx

```diff
@@ -387,24 +394,25 @@ const SitesDashboard = ( {
 						sites={ paginatedSites }
 						isLoading={ isLoading || ! initialSortApplied }
 						paginationInfo={ getSitesPagination( filteredSites, perPage ) }
-						dataViewsState={ dataViewsState }
-						setDataViewsState={ setDataViewsState }
+						viewState={ viewState }
+						setViewState={ setViewState }
+						selectedItem={ selectedSite as SiteExcerptData | null }
+						openSitePreviewPane={ openSitePreviewPane }
 					/>
 				</LayoutColumn>
 			) }
 
-			{ dataViewsState.selectedItem && (
+			{ selectedSite && (
 				<GuidedTourContextProvider
 					guidedTours={ onboardingTours }
 					preferenceNames={ CALYPSO_ONBOARDING_TOURS_PREFERENCE_NAME }
 					eventNames={ CALYPSO_ONBOARDING_TOURS_EVENT_NAMES }
 				>
 					<LayoutColumn className="site-preview-pane" wide>
 						<DotcomPreviewPane
-							site={ dataViewsState.selectedItem }
-							selectedSiteFeature={ selectedSiteFeature }
+							site={ selectedSite as SiteExcerptData }
```

Comment:
Need to figure out what SiteExcerptData is for, it might be a4a only. Otherwise need to come up with a casting strategy between SiteDetails and ExerptData

Reply:
It's a subset of SiteDetails, can add the undefined type and remove the type assertion.